src/api/app/views/webui/users/tokens/show.html.haml

```diff
@@ -1,3 +1,4 @@
+-# haml-lint:disable ViewLength
```

Comment:
Refactoring this view is tracked in our backlog [here](https://trello.com/c/6Oe3vBnC/214-split-up-the-users-tokens-showhtmlhaml-view).